src/sentry/static/sentry/app/views/releases/detail/releaseArtifacts.jsx

```diff
@@ -1,4 +1,4 @@
-import {Flex} from 'reflexbox';
+import {Flex} from 'reflexbox'; // eslint-disable-line no-restricted-imports
```

Comment:
Could we fix this instead of ignoring it 🙏

Reply:
I didn't plan on doing it initially as this entire component should be replaced with the new one in the near future.

But we know how that goes and it was a quick refactor, so here you go 😅